src/acceleration/Acceleration.hpp

```diff
@@ -64,9 +64,6 @@ class Acceleration {
   /// Checks if all dataIDs are contained in cplData
   void checkDataIDs(const DataMap &cplData) const;
 
-  /// Concatenates all coupling data involved into a single vector
```

Comment:
Please keep the documentation here and use `@copydoc` in the child classes. Compare [here](https://github.com/precice/precice/blob/15cdcbdf72fea5c06c996cc1c2e0f0b4e137916d/src/cplscheme/BiCouplingScheme.hpp#L91)